specification/cosmos-db/resource-manager/readme.go.md

```diff
@@ -21,6 +22,15 @@ batch:
   - tag: package-2019-08-preview
   - tag: package-2015-04
 ```
+### Tag: package-2020-09 and go
+
+These settings apply only when `--tag=package-2020-09 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2020-09' && $(go)
+output-folder: $(go-sdk-folder)/services/cosmos-db/mgmt/2020-09/$(namespace)
```

Comment:
We need to put it in preview sub-directory because this tag contains preview swaggers.
```suggestion
output-folder: $(go-sdk-folder)/services/preview/cosmos-db/mgmt/2020-09/$(namespace)
```